doc/sources/installation/installation-rpi.rst

```diff
@@ -107,42 +107,36 @@ Manual installation (On Raspbian Wheezy)
     echo "export PYTHONPATH=$(pwd):\$PYTHONPATH" >> ~/.profile
     source ~/.profile
 
-Manual installation (On Arch Linux ARM)
+Manual installation (On Arch Linux based systems)
```

Comment:
I'm tempted to ask to re-organize the page such that there's one subsection with multiple subsubsections listing the OS deps for Jessie/Wheezy/arch and a final subsection showing the kivy install part because they should be shared and the same for the OSs.

Only do it if you want to do it though.

Reply:
I wanted to change the name because there are many Arch based systems. Manjaro has 11 different images you can use.